kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/source/reader/ResultReader.scala

```diff
@@ -98,7 +98,7 @@ object ResultReader extends LazyLogging {
       inputStream <- storageInterface.getBlob(pathWithLine.bucket, path).leftMap(_.toException)
       fileSize    <- storageInterface.getBlobSize(pathWithLine.bucket, path).leftMap(_.toException)
       _ <- Try(logger.info(
-        s"[${connectorTaskId.show}] Reading next file: ${pathWithLine.show} from line ${pathWithLine.line}",
+        s"[${connectorTaskId.show}] Reading next file: ${pathWithLine.toPath} from line ${pathWithLine.line}",
```

Comment:
I notice the following:

S3Location.toPath is only ever used in loggers
S3Location.toString has been replaced with show
S3Location.fromStart is no longer used

Can these 3 methods be removed and only use `pathWithLine.show` going forward?

Reply:
Valid point, however if you look at the `show` method

```
    Show.show(loc => s"${loc.bucket}//:${loc.prefix.map(s => s"$s/").getOrElse("")}|${loc.path.getOrElse("")}")
```
It uses `|` and that does not make it quick to use. 

The `show` does not provide the full S3Location, and a toString should capture all info. Whereas we need most of the time the path only.

Reply:
I removed the path, but i think there needs to be a distinction between show and tostring, only because different info is required. Show is meant to be used as a better .tostring, and we invert the usage here, for now.